README.rst.j2

```diff
@@ -0,0 +1,160 @@
+This is a DRAFT (possibly including untested, yet-to-be released) edits to the blog post published here: https://www.caktusgroup.com/blog/2017/03/14/production-ready-dockerfile-your-python-django-app/
+
+**Have a suggested tweak or fix?** Feel free to submit a PR.
+
+
+A Production-ready Dockerfile for Your Python/Django App
+========================================================
+
+**Update (October 29, 2019):** I updated this post with more recent Django and Postgres versions, to use Python and pip directly in the container (instead of in a separate virtual environment, which was unnecessary), and switched to a non-root user via Docker instead of uWSGI.
+
+**Update (April 8, 2019):** I updated this post for Python 3.7 and to use the "slim" (Debian-based) Docker image. This version of my post includes a number of other minor improvements, including a less-magical approach to installing (and removing) system dependencies, better-documented uWSGI settings, a list of the requirements and settings you'll need to match this Dockerfile, support for static file serving via uWSGI (including hashed filenames and cache-forever headers), and an accompanying `GitHub repository <https://github.com/caktus/dockerfile_post/>`_.
+
+Docker has matured a lot since it was released. We've been watching it closely at Caktus, and have been thrilled by the adoption -- both by the community and by service providers. As a team of Python and Django developers, we're always searching for best of breed deployment tools. Docker is a clear fit for packaging the underlying code for many projects, including the Python and Django apps we build at Caktus.
+
+
+Technical Overview
+------------------
+
+There are many ways to containerize a Python/Django app, no one of which could be considered "the best." That being said, I think the following approach provides a good balance of simplicity, configurability, and container size. The specific tools I use are: `Docker <https://www.docker.com/>`_ (of course), the `python:3.7-slim <https://hub.docker.com/_/python/>`_ Docker image (based on Debian Stretch), and `uWSGI <https://uwsgi-docs.readthedocs.io/>`_.
```

Comment:
As long as we're updating this and Python 3.8 is already out, we could go ahead and use 3.8.

Reply:
Actually, it looks like 2.2 will never officially support 3.8: https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django